.github/workflows/build-packages.yml

```diff
@@ -84,64 +84,75 @@ jobs:
           # Copy the compiler to the expected location to avoid rebuilding it when creating the NuGet packages
           New-Item -ItemType Directory -Path "$env:GITHUB_WORKSPACE\tools\slicec-cs\target\release"
           Copy-Item "slicec-cs-x86_64-pc-windows-msvc\slicec-cs.exe" -Destination "$env:GITHUB_WORKSPACE\tools\slicec-cs\target\release"
+
       - name: Build IceRPC Tools
         working-directory: tools
         run: dotnet build --configuration Release
+
       - name: Build IceRPC
         run: dotnet build --configuration Release
+
+      - name: Collect Files to Sign
+        run: |
+            $assemblies = @(
+                'src\IceRpc.Extensions.DependencyInjection\bin\Release\net10.0\IceRpc.Extensions.DependencyInjection.dll'
```

Comment:
We can't use glob to find the src and tools files? This would be more maintainable.